File: doc/voorbeeld.pas
program voorbeeld(input, output);

var
   aantaltests, test : integer;
   naam				 : string[100];

begin
   readln(aantaltests);

   for test := 1 to aantaltests do
   begin
	  readln(naam);
	  writeln('Hallo ',naam,'!');
   end;
end.

File: doc/voorbeeld.cc
using namespace std;

#include <iostream>
#include <string>

int main()
{
    int aantaltests, test;
    string naam;

    cin >> aantaltests;

    for(test=1; test<=aantaltests; test++) {
        cin >> naam;
        cout << "Hallo " << naam << "!\n";
    }
    
    return 0;
}
